openapi.v2.json

```diff
@@ -6844,12 +7464,12 @@
                 },
                 "start_time": {
                     "type": "string",
-                    "format": "date-time",
+                    "format": "dateTime",
```

Comment:
the correct format is "date-time" https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#dataTypeFormat